src/NuGet.Core/NuGet.CommandLine.XPlat/XPlatUtility.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Net;
+using Microsoft.Dnx.Runtime.Common.CommandLine;
+using NuGet.Common;
+using NuGet.Logging;
+using NuGet.Protocol.Core.Types;
+
+namespace NuGet.CommandLine.XPlat
+{
+    public class XPlatUtility
+    {
+        internal const string HelpOption = "-h|--help";
+        internal const string VerbosityOption = "-v|--verbosity <verbosity>";
+        internal static readonly int MaxDegreesOfConcurrency = Environment.ProcessorCount;
```

Comment:
We could try different settings. Currently it matches DNU by using the processor count. --disable-parallel can be used to turn it off.